paasJobs/scripts/gc_crawl_then_upload.sh

```diff
@@ -124,8 +124,9 @@ function run_crawler_downloader() {
     -e "LOCAL_DOWNLOAD_DIRECTORY_PATH=${CRAWLER_CONTAINER_DL_DIR}" \
     -e "LOCAL_PREVIOUS_MANIFEST_LOCATION=${CRAWLER_CONTAINER_MANIFEST_LOCATION}" \
     -e "TEST_RUN=${TEST_RUN:-no}" \
-    "${CRAWLER_CONTAINER_IMAGE}" \
-  && docker cp "$container_name":"$CRAWLER_CONTAINER_DL_DIR" "$HOST_JOB_DL_DIR"
+    "${CRAWLER_CONTAINER_IMAGE}"
```

Comment:
let's put the `docker_run_status` right after the docker run command and make sleep 2 minutes (120), just to be sure